pkg/handler/flag_migrations.go

```diff
@@ -0,0 +1,215 @@
+package handler
+
+import (
+	"fmt"
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/openflagr/flagr/pkg/entity"
+	"github.com/openflagr/flagr/pkg/mapper/entity_restapi/r2e"
+	"github.com/openflagr/flagr/pkg/util"
+	"github.com/openflagr/flagr/swagger_gen/models"
+	"github.com/openflagr/flagr/swagger_gen/restapi/operations/migration"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cast"
+	"gorm.io/gorm/clause"
+	"os"
+	"sigs.k8s.io/yaml"
+	"strings"
+)
+
+type FlagMigrationOptions struct {
+	Run  bool   `long:"migrations" short:"m" description:"Run Flag Migrations and Exit"`
+	Path string `long:"migrationPath" description:"Migration files path" env:"FLAGR_MIGRATION_PATH"`
+}
+
+func FlagMigrations(path string) (string, error) {
+	return migrateFromDir(path)
+}
+var runMigrationsHandler = func(migration.RunMigrationsParams) middleware.Responder {
+	logrus.Info("Running migrations")
+	migrationPath, hasMigrationPath := os.LookupEnv("FLAGR_MIGRATION_PATH")
```

Comment:
let's move all the config related env to the env.go, so that it's auto documented